easybatch-integration/easybatch-apache-commons-csv/src/main/java/org/easybatch/integration/apache/common/csv/ApacheCommonCsvRecordMapper.java

```diff
@@ -40,7 +43,11 @@
     private ObjectMapper<T> objectMapper;
 
     public ApacheCommonCsvRecordMapper(Class<? extends T> recordClass) {
-        this.objectMapper = new ObjectMapper<T>(recordClass);
+        this(recordClass, true);
+    }
+
+    public ApacheCommonCsvRecordMapper(Class<? extends T> recordClass, boolean emptyToNull) {
```

Comment:
I don't think it should go to TypeConverter. Type converters should be as simple as possible.

In fact, it's really related to your comment here : https://github.com/EasyBatch/easybatch-framework/pull/105#issuecomment-127681812. 

The `emptyToNull` could be added to the abstract class you have mentionned in your comment. Maybe this abstract class will simply extends the ObjectMapper class.